package/docker/scripts/docker_build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -xe
+
+# Build Artifact `implementer-interface.zip` should be placed in dist directory in the root of repository
+
+#Building Docker images
+IMPLEMENTER_INTERFACE_IMAGE_TAG=${BAHMNI_VERSION:?}-${GITHUB_RUN_NUMBER:?}
+docker build -t bahmni/implementer-interface:${IMPLEMENTER_INTERFACE_IMAGE_TAG} -f package/docker/Dockerfile  . --no-cache
```

Comment:
the build script should be coherent to create a docker image from when it was triggered... its dependent on a dist to be present - and the dist could be a dirty one i.e. may or may-not have the latest binaries. You can add a `yarn ci` before docker build...